strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+| 3 | a. | DCP accession numbers, DOIs and other means of citation of DCP data are more prevalent in publications than the corresponding submission at an additional resource |
+
+
+### Aim 2
+
+**Provide researchers access to data that will enable the Human Cell Atlas to answer fundamental questions in all aspects of biology.**
+
+#### Value Assumptions
+
+The two main types of scientists targeted by the DCP (computational biologists and “wet-lab” researchers in smaller groups) will search for data of interest using a limited, focused number of relatively simple characteristics before running their chosen analysis tools on that set of data.
+
+Scientists will utilise data from the DCP as long as it is easy to access. If metadata they need are missing, or if it is difficult to access and work with DCP data, they will try to source the data from elsewhere.
+
+Scientists will access analysed data that has been generated in their area of research interest in order to exploit it using trusted analysis tools.
+
+Scientists will explore the data in the DCP via a user interface in order to find data relevant to their research interest, before using programmatic techniques and tools to perform more in-depth analyses. They expect the DCP to provide a consistent experience across user interfaces and APIs.
+
+In order for it to be easy to use or develop analysis tools, data in the DCP must be easy to use and the experience of discovering and using this data must be the same whether data is accessed via user interfaces or APIs.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | The same scientists return to the DCP to access data as new datasets are published (i.e. the DCP is retaining existing scientists because the data is useable) |
+|   | b. | Analysed data is accessed more commonly than raw data |
+|   | c. | User requests for additions or modifications to the metadata available are relatively rare |
+| 2 | a. | User feedback and UX research indicates a high degree of satisfaction with DCP user interfaces and APIs |
+| 3 | a. | Relative usage of the UI for data searches, and the API for direct data access, show consistent patterns over time, rather than changing significantly depending on which access mode is easier |
+|   | b. | Search analytics show datasets are discovered using a small number of key searches common to a wide number of scientists |
+|   | c. | All data used in the Human Cell Atlas or tissue-specific cellular atlases (e.g. brain atlas) is present in the DCP |
+
+
+### Aim 3
+
+**Perform regular data releases of high value to single cell researchers.**
+
+#### Value Assumptions
+
+The two main types of scientists, when consuming others’ data from the platform, value consistency of data and metadata, in terms of standards, versions and quality control over the speed of access.
```

Comment:
You mean researchers with a pipette vs. keyboard?  I would specifically mention these.